community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -451,6 +461,20 @@ public static void main( String[] incomingArguments, boolean defaultSettingsSuit
         }
     }
 
+    private static Long parseMaxMemory( String maxMemoryString )
+    {
+        if ( maxMemoryString != null )
+        {
+            if ( maxMemoryString.endsWith( "%" ) )
```

Comment:
we at least need to trim user provided string before trying to figure out what is inside of it